dashboard/test/models/concerns/partial_registration_test.rb

```diff
@@ -109,10 +105,13 @@ class PartialRegistrationTest < ActiveSupport::TestCase
     assert_kind_of User, User.new_from_partial_registration(@session)
   end
 
-  test 'new_from_partial_registration does not save the User' do
+  test 'new_with_session does not save the User' do
```

Comment:
Why change the method being tested here? Was this just an improvement so that we cover the addition of email to the session attributes?

Reply:
`new_from_partial_registration` by itself isn't sufficient to create a user anymore as you need to feed the email in. `new_with_session` does exactly that and calls `new_from_partial_registration` with the parameters so it made more sense to use `new_with_session` as it still tests partial registration with the extra information needed.